app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,2 @@
+class TeamsController < ApplicationController
```

Comment:
Specifically, why now? I know you'll eventually need this, but what does it have to do with this commit?